store/store.go

```diff
@@ -146,6 +152,13 @@ func myappsURL() string {
 	return "https://myapps.developer.ubuntu.com/api/2.0"
 }
 
+func scaURL() string {
+	if useStagingStore() {
```

Comment:
gah, sorry @pete-woods,  not that having all those flags is any good but probably not an issue with your branch either, anyway @chipaca is right on this one